solr/contrib/analytics/src/java/org/apache/solr/analytics/util/MedianCalculator.java

```diff
@@ -35,7 +35,9 @@
     select(list, .5 * size, 0, size);
 
     int firstIdx = (int) (Math.floor(.5 * size));
-    int secondIdx = (firstIdx <= size && size % 2 == 1) ? firstIdx + 1 : firstIdx;
+    // from findbug bug info: 
+    // The code uses x % 2 == 1 to check to see if a value is odd, but this won't work for negative numbers (e.g., (-5) % 2 == -1). If this code is intending to check for oddness, consider using x & 1 == 1, or x % 2 != 0. 
+    int secondIdx = (firstIdx <= size && size % 2 != 0) ? firstIdx + 1 : firstIdx;
```

Comment:
@HoustonPutman have a look at this. I'll not merge this PR but perhaps you like to fix this one bug in a separate commit?

Reply:
This is a no-op since size is guaranteed to be non-negative. Also this analytics code is extremely out of date, and has pretty much all been re-written since this PR.